### PR TITLE
[FEAT] add Issue template

### PR DESCRIPTION
**Pull request by hochan222**
As the issue was raised, I received an inquiry because the form was not set to report the problem. https://github.com/mdn/translated-content/issues/907

We really needed some templates to help people file issues. I imported the issue template of the content repo. https://github.com/mdn/content/pull/2906

How about adding it?

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@mdn/core-yari-dev is it possible to review this?

File: .github/ISSUE_TEMPLATE/content-bug.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Content bug
```
Comment (peterbe) [inline .github/ISSUE_TEMPLATE/content-bug.md:2]: Should we make it 

```suggestion
name: Translated content bug
```
?

**Review by peterbe — APPROVED** [review]
Actually, if this verbatim copy of mdn/content let's do it!

File: .github/ISSUE_TEMPLATE/content-bug.md (hunk 1 above)
Comment (hochan222) [reply .github/ISSUE_TEMPLATE/content-bug.md:2]: Oh I thought it was a word meaning content, but it was a repository name! But you merged, I guess I was right!

**Pull request merged**